contracts/lper/src/msg.rs

```diff
@@ -0,0 +1,34 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct InstantiateMsg {
+    pub lp_position: LPInfo,
+    pub clock_address: String,
+    pub holder_address: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct LPInfo {
+    pub addr: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ExecuteMsg {
+    Tick {},
+    WithdrawRewards {},
```

Comment:
This should be called withdraw liquidity. Keep same semantics as Astroport
```suggestion
    WithdrawLiquidity {},
```